control-plane/agents/src/bin/core/frontend_node/spec.rs

```diff
@@ -0,0 +1,48 @@
+use crate::controller::resources::operations_helper::{ResourceSpecs, ResourceSpecsLocked};
+use grpc::operations::{PaginatedResult, Pagination};
+use stor_port::types::v0::store::frontend_node::FrontendNodeSpec;
+
+impl ResourceSpecs {
+    /// Get all frontend node specs.
```

Comment:
Just to make it clear, and inline with existing ones.

```suggestion
    /// Get a copy of all frontend node specs.
```